rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.79.0"
```

Comment:
Same bump to `1.80.1` possible here.

Reply:
Good point, 7fe48c9

Reply:
I'm not quite sure why the commit is named _bump to 0.80.1_, but other than that, this comment is resolved.

Reply:
Hupsi, close enough ^^